.github/workflows/task-test.yml

```diff
@@ -86,7 +90,10 @@ jobs:
             tests/pytests/requirements.*
       - name: Setup specific
         working-directory: setup/.install
-        run: ./install_script.sh ${{ steps.mode.outputs.mode }}
+        run: |
+          ./install_script.sh ${{ steps.mode.outputs.mode }}
+          ./install_llvm.sh ${{ steps.mode.outputs.mode }} ${{ env.LLVM_VERSION }}
+          ./install_rust.sh 
```

Comment:
As I see it, installing rust is not part of the setup, but rather part of the test setup (it is required for testing against JSON only). I think it is better to separate between the two groups of dependencies